jsx/components/NoMatch.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+const NoMatch = () => {
+  return (
+    <p>You've hit our 404 page. <a href="/">Go home</a>.</p>
```

Comment:
Thou shalt "Go home"!

LUL

Reply:
Maybe I should add some styles to this page to give it a little more love visually.